Auto update failing for vulkan-extension-layer

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-extension-layer](https://github.com/termux/termux-packages/blob/5f9fd1361ec4a005f21c7b7618cddd83e3bb367f/packages/vulkan-extension-layer) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-extension-layer [Current version: 1.3.298]
INFO: package being updated to 1.3.299.
Downloading https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v1.3.299.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  771k    0  771k    0     0  2078k      0 --:--:-- --:--:-- --:--:-- 2078k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-extension-layer/build.sh b/packages/vulkan-extension-layer/build.sh
index 99ca936cb1..54c4bcbd40 100644
--- a/packages/vulkan-extension-layer/build.sh
+++ b/packages/vulkan-extension-layer/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ExtensionLayer
 TERMUX_PKG_DESCRIPTION="Vulkan Extension Layer"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.298"
+TERMUX_PKG_VERSION="1.3.299"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=94a8020410bfc79d3b27c0f6c84bd25e0c24ae47ea02fd4d7c15ceec5bd57b2b
+TERMUX_PKG_SHA256=cc2944bcc2034b56a435e156229636ed8a8141ef68c0056582ab7c19c2af9b90
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-extension-layer): 1.3.299


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-extension-layer exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-extension-layer for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100 13134  100 13134    0     0  10893      0  0:00:01  0:00:01 --:--:-- 10899
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3363      0 --:--:-- --:--:-- --:--:--  3358
100   833  100   833    0     0   3362      0 --:--:-- --:--:-- --:--:--  3358
gpg: Signature made Thu 24 Oct 2024 11:45:16 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  80935      0 --:--:-- --:--:-- --:--:-- 81219
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5254      0 --:--:-- --:--:-- --:--:--  5272
gpg: Signature made Thu 24 Oct 2024 05:53:35 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  71079      0 --:--:-- --:--:-- --:--:-- 71304
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5083      0 --:--:-- --:--:-- --:--:--  5110
gpg: Signature made Thu 24 Oct 2024 11:46:39 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_27b_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  291k  100  291k    0     0  4497k      0 --:--:-- --:--:-- --:--:-- 4555k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-support/libandroid-support_29_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10852  100 10852    0     0   170k      0 --:--:-- --:--:-- --:--:--  170k
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.3 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libexpat/libexpat_2.6.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 85624  100 85624    0     0   846k      0 --:--:-- --:--:-- --:--:--  853k
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.6-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30640  100 30640    0     0   406k      0 --:--:-- --:--:-- --:--:--  409k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  8083k      0 --:--:-- --:--:-- --:--:-- 7990k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0   113k      0 --:--:-- --:--:-- --:--:--  113k
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.3 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.6.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  184k  100  184k    0     0  2738k      0 --:--:-- --:--:-- --:--:-- 2751k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.3 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.6.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 66952  100 66952    0     0   755k      0 --:--:-- --:--:-- --:--:--  760k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  525k  100  525k    0     0  7381k      0 --:--:-- --:--:-- --:--:-- 7399k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87728  100 87728    0     0  1325k      0 --:--:-- --:--:-- --:--:-- 1338k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32964  100 32964    0     0   392k      0 --:--:-- --:--:-- --:--:--  397k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39644  100 39644    0     0   640k      0 --:--:-- --:--:-- --:--:--  645k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/readline/readline_8.2.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  228k  100  228k    0     0  2101k      0 --:--:-- --:--:-- --:--:-- 2115k
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.24_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  139k  100  139k    0     0  2295k      0 --:--:-- --:--:-- --:--:-- 2332k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0  65069      0 --:--:-- --:--:-- --:--:-- 65523
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26634  100 26634    0     0   512k      0 --:--:-- --:--:-- --:--:--  520k
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 25506  100 25506    0     0   483k      0 --:--:-- --:--:-- --:--:--  488k
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.09.24 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2024.09.24_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  125k  100  125k    0     0  1635k      0 --:--:-- --:--:-- --:--:-- 1653k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.09.24 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2024.09.24_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  114k  100  114k    0     0  1811k      0 --:--:-- --:--:-- --:--:-- 1824k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.3.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2261k  100 2261k    0     0  22.2M      0 --:--:-- --:--:-- --:--:-- 22.3M
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.3.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  255k  100  255k    0     0  2932k      0 --:--:-- --:--:-- --:--:-- 2936k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcrypt/libcrypt_0.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8720  100  8720    0     0   178k      0 --:--:-- --:--:-- --:--:--  177k
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tcl/tcl_8.6.14_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2209k  100 2209k    0     0  24.1M      0 --:--:-- --:--:-- --:--:-- 24.2M
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.46.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite/libsqlite_3.46.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  596k  100  596k    0     0  7204k      0 --:--:-- --:--:-- --:--:-- 7270k
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.46.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/sqlite/sqlite_3.46.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  566k  100  566k    0     0  6599k      0 --:--:-- --:--:-- --:--:-- 6669k
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.46.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite-tcl/libsqlite-tcl_3.46.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 17436  100 17436    0     0   318k      0 --:--:-- --:--:-- --:--:--  315k
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/brotli/brotli_1.1.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  323k  100  323k    0     0  4829k      0 --:--:-- --:--:-- --:--:-- 4895k
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.44 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpng/libpng_1.6.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  191k  100  191k    0     0  3183k      0 --:--:-- --:--:-- --:--:-- 3240k
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/freetype/freetype_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  399k  100  399k    0     0  4934k      0 --:--:-- --:--:-- --:--:-- 4999k
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/ttf-dejavu/ttf-dejavu_2.37-8_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2410k  100 2410k    0     0  24.6M      0 --:--:-- --:--:-- --:--:-- 24.7M
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig/fontconfig_2.15.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  119k  100  119k    0     0  1701k      0 --:--:-- --:--:-- --:--:-- 1704k
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig-utils/fontconfig-utils_2.15.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14532  100 14532    0     0   195k      0 --:--:-- --:--:-- --:--:--  197k
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.1 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorg-util-macros/xorg-util-macros_1.20.1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21908  100 21908    0     0   275k      0 --:--:-- --:--:-- --:--:--  277k
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorgproto/xorgproto_2024.1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  243k  100  243k    0     0  3415k      0 --:--:-- --:--:-- --:--:-- 3432k
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxau/libxau_1.0.11_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8488  100  8488    0     0  71913      0 --:--:-- --:--:-- --:--:-- 72547
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxdmcp/libxdmcp_1.1.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24352  100 24352    0     0   412k      0 --:--:-- --:--:-- --:--:--  417k
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xcb-proto/xcb-proto_1.17.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 78616  100 78616    0     0  1391k      0 --:--:-- --:--:-- --:--:-- 1395k
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-xcbgen/python-xcbgen_1.17.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 44572  100 44572    0     0   746k      0 --:--:-- --:--:-- --:--:--  750k
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxcb/libxcb_1.17.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  913k  100  913k    0     0  10.5M      0 --:--:-- --:--:-- --:--:-- 10.6M
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.1 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xtrans/xtrans_1.5.1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39596  100 39596    0     0  72178      0 --:--:-- --:--:-- --:--:-- 72123
100 39596  100 39596    0     0  72166      0 --:--:-- --:--:-- --:--:-- 72123
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.10 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libx11/libx11_1.8.10_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1868k  100 1868k    0     0  22.2M      0 --:--:-- --:--:-- --:--:-- 22.5M
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrender/libxrender_0.9.11_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 23576  100 23576    0     0   473k      0 --:--:-- --:--:-- --:--:--  479k
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxft/libxft_2.3.8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 47896  100 47896    0     0   730k      0 --:--:-- --:--:-- --:--:--  742k
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxext/libxext_1.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 97968  100 97968    0     0  1561k      0 --:--:-- --:--:-- --:--:-- 1568k
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxss/libxss_1.2.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11924  100 11924    0     0   229k      0 --:--:-- --:--:-- --:--:--  232k
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tk/tk_8.6.14_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1715k  100 1715k    0     0  20.3M      0 --:--:-- --:--:-- --:--:-- 20.4M
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.7-1 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.12.7-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7399k  100 7399k    0     0  71.6M      0 --:--:-- --:--:-- --:--:-- 72.2M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.7-1 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.12.7-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1190k  100 1190k    0     0  14.1M      0 --:--:-- --:--:-- --:--:-- 14.1M
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.7-1 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.12.7-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1733k  100 1733k    0     0  14.2M      0 --:--:-- --:--:-- --:--:-- 14.3M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.13.4-1 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.13.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  456k  100  456k    0     0  6376k      0 --:--:-- --:--:-- --:--:-- 6431k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.4-1 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.13.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30892  100 30892    0     0   560k      0 --:--:-- --:--:-- --:--:--  569k
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.4-1 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.13.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  189k  100  189k    0     0  2967k      0 --:--:-- --:--:-- --:--:-- 3000k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.4-1 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.13.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 73836  100 73836    0     0  1333k      0 --:--:-- --:--:-- --:--:-- 1360k
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.22.0-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.22.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0  1652k      0 --:--:-- --:--:-- --:--:-- 1667k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.22.0-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.22.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 81144  100 81144    0     0  1427k      0 --:--:-- --:--:-- --:--:-- 1440k
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-headers@1.3.299 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.3.299_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1352k  100 1352k    0     0  16.8M      0 --:--:-- --:--:-- --:--:-- 16.7M
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.3.299 if necessary...
Found vulkan-utility-libraries in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-utility-libraries/vulkan-utility-libraries_1.3.299_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 39  485k   39  189k    0     0   245k      0  0:00:01 --:--:--  0:00:01  245k
100  485k  100  485k    0     0   554k      0 --:--:-- --:--:-- --:--:--  554k
extracting vulkan-utility-libraries to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v1.3.299.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  771k    0  771k    0     0  2181k      0 --:--:-- --:--:-- --:--:-- 2181k
Applying patch: 0001-no-android.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The C compiler identification is Clang 18.0.2
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done (0.4s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-extension-layer/build
[1/12] Building CXX object utils/CMakeFiles/VkExtLayer_utils.dir/allocator.cpp.o
[2/12] Building CXX object layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/generated/shader_object_full_draw_state_struct_members.cpp.o
FAILED: layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/generated/shader_object_full_draw_state_struct_members.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -DVkLayer_khronos_shader_object_EXPORTS -I/home/builder/.termux-build/vulkan-extension-layer/src/layers/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/generated -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-unused-parameter -Wno-missing-field-initializers -Wno-implicit-int-conversion -Wno-sign-conversion -Wno-string-conversion -Wno-implicit-fallthrough -Wno-implicit-int-float-conversion -Wno-shorten-64-to-32 -MD -MT layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/generated/shader_object_full_draw_state_struct_members.cpp.o -MF layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/generated/shader_object_full_draw_state_struct_members.cpp.o.d -o layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/generated/shader_object_full_draw_state_struct_members.cpp.o -c /home/builder/.termux-build/vulkan-extension-layer/src/layers/shader_object/generated/shader_object_full_draw_state_struct_members.cpp
In file included from /home/builder/.termux-build/vulkan-extension-layer/src/layers/shader_object/generated/shader_object_full_draw_state_struct_members.cpp:20:
In file included from /home/builder/.termux-build/vulkan-extension-layer/src/layers/./shader_object/shader_object_structs.h:29:
/data/data/com.termux/files/usr/include/vulkan/vulkan.h:71:10: fatal error: 'X11/extensions/Xrandr.h' file not found
   71 | #include <X11/extensions/Xrandr.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[3/12] Building CXX object layers/CMakeFiles/VkLayer_khronos_synchronization2.dir/synchronization2.cpp.o
FAILED: layers/CMakeFiles/VkLayer_khronos_synchronization2.dir/synchronization2.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -DVkLayer_khronos_synchronization2_EXPORTS -I/home/builder/.termux-build/vulkan-extension-layer/src/layers/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/generated -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-unused-parameter -Wno-missing-field-initializers -Wno-implicit-int-conversion -Wno-sign-conversion -Wno-string-conversion -Wno-implicit-fallthrough -Wno-implicit-int-float-conversion -Wno-shorten-64-to-32 -MD -MT layers/CMakeFiles/VkLayer_khronos_synchronization2.dir/synchronization2.cpp.o -MF layers/CMakeFiles/VkLayer_khronos_synchronization2.dir/synchronization2.cpp.o.d -o layers/CMakeFiles/VkLayer_khronos_synchronization2.dir/synchronization2.cpp.o -c /home/builder/.termux-build/vulkan-extension-layer/src/layers/synchronization2.cpp
In file included from /home/builder/.termux-build/vulkan-extension-layer/src/layers/synchronization2.cpp:23:
In file included from /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.h:16:
/data/data/com.termux/files/usr/include/vulkan/vulkan.h:71:10: fatal error: 'X11/extensions/Xrandr.h' file not found
   71 | #include <X11/extensions/Xrandr.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[4/12] Building CXX object layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/shader_object.cpp.o
FAILED: layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/shader_object.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -DVkLayer_khronos_shader_object_EXPORTS -I/home/builder/.termux-build/vulkan-extension-layer/src/layers/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/generated -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-unused-parameter -Wno-missing-field-initializers -Wno-implicit-int-conversion -Wno-sign-conversion -Wno-string-conversion -Wno-implicit-fallthrough -Wno-implicit-int-float-conversion -Wno-shorten-64-to-32 -MD -MT layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/shader_object.cpp.o -MF layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/shader_object.cpp.o.d -o layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object/shader_object.cpp.o -c /home/builder/.termux-build/vulkan-extension-layer/src/layers/shader_object/shader_object.cpp
In file included from /home/builder/.termux-build/vulkan-extension-layer/src/layers/shader_object/shader_object.cpp:32:
/data/data/com.termux/files/usr/include/vulkan/vulkan.h:71:10: fatal error: 'X11/extensions/Xrandr.h' file not found
   71 | #include <X11/extensions/Xrandr.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[5/12] Building CXX object layers/CMakeFiles/VkLayer_khronos_memory_decompression.dir/decompression/decompression.cpp.o
FAILED: layers/CMakeFiles/VkLayer_khronos_memory_decompression.dir/decompression/decompression.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -DVkLayer_khronos_memory_decompression_EXPORTS -I/home/builder/.termux-build/vulkan-extension-layer/src/layers/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/. -I/home/builder/.termux-build/vulkan-extension-layer/src/utils/generated -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-unused-parameter -Wno-missing-field-initializers -Wno-implicit-int-conversion -Wno-sign-conversion -Wno-string-conversion -Wno-implicit-fallthrough -Wno-implicit-int-float-conversion -Wno-shorten-64-to-32 -MD -MT layers/CMakeFiles/VkLayer_khronos_memory_decompression.dir/decompression/decompression.cpp.o -MF layers/CMakeFiles/VkLayer_khronos_memory_decompression.dir/decompression/decompression.cpp.o.d -o layers/CMakeFiles/VkLayer_khronos_memory_decompression.dir/decompression/decompression.cpp.o -c /home/builder/.termux-build/vulkan-extension-layer/src/layers/decompression/decompression.cpp
In file included from /home/builder/.termux-build/vulkan-extension-layer/src/layers/decompression/decompression.cpp:31:
In file included from /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.h:16:
/data/data/com.termux/files/usr/include/vulkan/vulkan.h:71:10: fatal error: 'X11/extensions/Xrandr.h' file not found
   71 | #include <X11/extensions/Xrandr.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-10-25 00:29:25 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/11509551433">11509551433</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-extension-layer/build.sh
+++ b/packages/vulkan-extension-layer/build.sh
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ExtensionLayer
 TERMUX_PKG_DESCRIPTION="Vulkan Extension Layer"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.298"
+TERMUX_PKG_VERSION="1.3.299"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=94a8020410bfc79d3b27c0f6c84bd25e0c24ae47ea02fd4d7c15ceec5bd57b2b
+TERMUX_PKG_SHA256=cc2944bcc2034b56a435e156229636ed8a8141ef68c0056582ab7c19c2af9b90
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
-TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
+TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"

